### PR TITLE
Added index table values to oca dataset

### PR DESCRIPTION
**Pull request by Tim-Jackins**

### CODE REVIEW

File: datasets/migrations/0094_auto_20210916_2301.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        ),
+        migrations.AddField(
+            model_name='ocahousingcourt',
+            name='court',
+            field=models.TextField(blank=True, null=True),
+        ),
+        migrations.AddField(
+            model_name='ocahousingcourt',
+            name='dateofjurydemand',
+            field=models.TextField(blank=True, null=True),
```
Comment (genceyalcin) [inline datasets/migrations/0094_auto_20210916_2301.py:26]: Should this be a datefield?

File: datasets/models/OCAHousingCourt.py [hunk 2]
```diff
@@ -57,6 +57,17 @@ class Meta:
...
+    court = models.TextField(blank=True, null=True)
+    fileddate = models.DateField(blank=True, null=True)
+    propertytype = models.TextField(blank=True, null=True)
+    classification = models.TextField(blank=True, null=True)
+    specialtydesignationtypes = models.TextField(blank=True, null=True)
+    disposeddate = models.DateField(blank=True, null=True)
+    disposedreason = models.TextField(blank=True, null=True)
+    firstpaper = models.TextField(blank=True, null=True)
+    primaryclaimtotal = models.DecimalField(decimal_places=8, max_digits=16, blank=True, null=True)
+    dateofjurydemand = models.TextField(blank=True, null=True)
```
Comment (genceyalcin) [inline datasets/models/OCAHousingCourt.py:69]: Should be a date?

Comment (genceyalcin) [reply datasets/models/OCAHousingCourt.py:69]: https://github.com/housing-data-coalition/oca/tree/master/docs

File: datasets/models/OCAHousingCourt.py [hunk 3]
```diff
@@ -57,6 +57,17 @@ class Meta:
...
+
```
Comment (genceyalcin) [inline datasets/models/OCAHousingCourt.py:70]: Missing status from this list

File: datasets/migrations/0094_auto_20210916_2301.py (hunk 1 above)
Comment (Tim-Jackins) [reply datasets/migrations/0094_auto_20210916_2301.py:26]: The data we have is either empty or inconsistent for the datefield to be used.

File: datasets/models/OCAHousingCourt.py (hunk 2 above)
Comment (Tim-Jackins) [reply datasets/models/OCAHousingCourt.py:69]: The data we have is either empty or inconsistent for the datefield to be used.

**Review by genceyalcin — APPROVED** [review]

**Pull request merged**